### ISSUE TITLE
[Question] Silent PN (SPN)

### ISSUE THREAD

**Issue by dittmarconsulting** [appfeel/node-pushnotifications#56]
Hi there,

Is there a way to send silent push notifications with this library?

According to Apple (see below) the prop contentAvailable has to be set to 1 (i.e. `contentAvailable: 1`) for SPN to work but this doesn't have any effect.

```
Configuring a Silent Notification

The aps dictionary can also contain the content-available property. The content- available property with a value of 1 lets the remote notification act as a silent notification. When a silent notification arrives, iOS wakes up your app in the background so that you can get new data from your server or do background information processing. Users aren’t told about the new or changed information that results from a silent notification, but they can find out about it the next time they open your app.

For a silent notification, take care to ensure there is no alert, sound, or badge payload in the aps dictionary. If you don’t follow this guidance, the incorrectly-configured notification might be throttled and not delivered to the app in the background, and instead of being silent is displayed to the user
```

**Comment by alex-friedl** [comment]
Hello @dittmarconsulting,
currently there isn't but we are waiting for #42 and #49 to be merged in order to add support for silent push notifications via APN and FCM

**Comment by leograde** [comment]
It's already March 2018, any idea when one of these two solutions will be merged?

**Comment by alex-friedl** [comment]
Hello @dittmarconsulting , @leograde ,

could you have a look at #83, test the branch and give me feedback if this works for you? 
Unfortunately I don't have a working iOS setup.

Thank you!

**Comment by alex-friedl** [comment]
I merged #83. Silent push notifications should work as expected now. Please re-open the issue if anything unexpected comes up.
